code/orders-api-rest/api/openapi-rest.yml

```
@@ -806,7 +806,128 @@
         '401':
           description: Unauthorized - authentication required
+          
+    /v1/password-reset:
+    post:
+      tags:
+        - PasswordResetController
+      summary: Request password reset
+      description: |
+        Accepts an email and returns 200 status regardless of whether the email exists.
+        This prevents email enumeration attacks while maintaining user experience.
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)